src/components/Main.js

```diff
@@ -1,57 +1,64 @@
 import React from 'react';
 // import MechanismNode from '../model/nodes/mechanism/MechanismNode';
 import { withStyles } from '@mui/styles';
-import BG from "../assets/svg/bg-dotted.svg";
+import BG from '../assets/svg/bg-dotted.svg';
 import ModelInterpreter from '../model/Interpreter';
-import MetaDiagram, { ComponentsMap } from "@metacell/meta-diagram";
+import MetaDiagram, { ComponentsMap } from '@metacell/meta-diagram';
 import CustomLinkWidget from './views/projections/CustomLinkWidget';
 import GenericMechanism from './views/mechanisms/GenericMechanism';
 import { buildModel } from '../model/utils';
 import { PNLClasses } from '../constants';
-import CompositionDrawer from './views/CompositionDrawer';
+import { Sidebar } from './views/Sidebar/Sidebar';
 
 const mockModel = require('../resources/model').mockModel;
 
 const styles = () => ({
   root: {
     height: 'calc(100vh - 3.5rem)',
     width: '100%',
+    margin: 'none',
```

Comment:
@emekauja margin and padding values are not working, also i don't see the value add even id they were.